app/controllers/api/v0/admin/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V0
+    module Admin
+      class UsersController < ApiController
+        before_action :authenticate_with_api_key_or_current_user!
+        before_action :authorize_super_admin
+        skip_before_action :verify_authenticity_token, only: %i[create]
+
+        def create
+          User.invite!(user_params)
+
+          head :ok
+        end
+
+        private
+
+        def user_params
+          { email: params.require(:email), name: params[:name] }.compact_blank
```

Comment:
Given that we expect creators to use tools (e.g. their existing SSO, Zapier, etc) to post to this endpoint I wanted to keep the param structure as simple and flat as possible, hence slightly more manual param handling. We could of course also default the name to the email address but `User.invite!` does not insist on a name.